maintainers/scripts/regenerate-hackage-packages.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p coreutils
```

Comment:
I'm not super familiar with the nix-shell scripts in nixpkgs, but shouldn't this ideally pass `-I nixpkgs=../../.` to `nix-shell` here so that a known version of `bash` and `coreutils` are being used?

Similarly, `hackage2nix` and `hackage` could probably be pulled from nixpkgs in a similar fashion.  No need to explicitly call `nix-build` in the script, right?

Reply:
I think nix-shell only works for changing the environment, so I don't think we can use it for all-cabal-hashes/hackage (as it doesn't provide any binaries or other things that can in there).
I adopted it for all other uses though. I did not find any rule specifying that maintainer scripts shall be executed with a specific working directory (the repo root seems to be the common assumption), maybe that should be noted somewhere?